src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -43,44 +38,180 @@
           "type": "string"
         },
         "executable": {
-          "format": "uri",
           "type": "string"
         },
-        "fd_ufs": {
-          "$ref": "#/$defs/updatable_config"
-        },
         "field_table": {
-          "$ref": "#/$defs/updatable_config"
+          "additionalProperties": false,
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "minProperties": 1,
+              "patternProperties": {
+                "^.*$": {
+                  "additionalProperties": false,
+                  "properties": {
+                    "longname": {
+                      "type": "string"
+                    },
+                    "profile_type": {
+                      "oneOf": [
+                        {
+                          "additionalProperties": false,
+                          "properties": {
+                            "name": {
+                              "const": "fixed"
+                            },
+                            "surface_value": {
+                              "type": "number"
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "surface_value"
+                          ],
+                          "type": "object"
+                        },
+                        {
+                          "additionalProperties": false,
+                          "properties": {
+                            "name": {
+                              "const": "profile"
+                            },
+                            "surface_value": {
+                              "type": "number"
+                            },
+                            "top_value": {
+                              "type": "number"
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "surface_value",
+                            "top_value"
+                          ],
+                          "type": "object"
+                        }
+                      ]
+                    },
+                    "units": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "longname",
+                    "profile_type",
+                    "units"
+                  ],
+                  "type": "object"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "required": [
+            "base_file"
+          ],
+          "type": "object"
```

Comment:
A `field_table` file is going to be required either here or in the static files.

Reply:
Not according to the code, in `create_field_table():
```
config_values=self._config.get("field_table", {})
```

Should that become
```
config_values=self._config("field_table")
```

? Then it would be required. I can make that update if so.

Reply:
So, FV3 requires a `field_table` file to run. The user might want to be able to generate it from a YAML config, but may instead provide a fully formed one that needs no modification. In the first case, there would be a `field_table:` YAML, entry. In the second case, it would be provided as a static file entry.

Reply:
OK, so it sounds like `field_table` requires _at least one of_ `base_file` and `update_values`.

Reply:
I'll push what I think is a correct implementation of the above momentarily.

Reply:
Done. Please un-resolve this or create a new comment if there appear to be issues.